iocore/net/P_UDPPacket.h

```diff
@@ -246,6 +246,41 @@ new_incoming_UDPPacket(struct sockaddr *from, char *buf, int len)
 }
 
 TS_INLINE UDPPacket *
+new_incoming_UDPPacket(struct sockaddr *from, IOBufferBlock *block, int len)
```

Comment:
It seems like this one and the original one are not used anymore. Should we keep them?

Reply:
remove it.